### PR TITLE
feat: v4 subgraph on Sepelia in routing cron

### PR DESCRIPTION
**Pull request by jsy1218**
This is the minimum change in routing-api to support download v4 subgraph. I already verified in my local aws account to see the v4 subgraph getting downloaded:
Screenshot 2024-08-08 at 5 56 14 PM

### CODE REVIEW

**Comment by jsy1218** [comment]
* **#808** <a href="https://app.graphite.dev/github/pr/Uniswap/routing-api/808?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @jsy1218 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by mikeki — APPROVED** [review]

File: lib/cron/cache-config.ts [hunk 1]
```diff
@@ -54,6 +63,9 @@ export const v2SubgraphUrlOverride = (chainId: ChainId) => {
   }
 }
 
+const v4TrackedEthThreshold = 0 // Pools need at least 0 of trackedEth to be selected
```
Comment (xrsv) [inline lib/cron/cache-config.ts:66]: are those final values? if not, maybe add a TODO

File: lib/handlers/marshalling/route-marshaller.ts [hunk 2]
```diff
@@ -73,17 +101,26 @@ export class RouteMarshaller {
...
+          TokenMarshaller.unmarshal(v4Route.output)
+        )
       case Protocol.MIXED:
         const mixedRoute = marshalledRoute as MarshalledMixedRoute
         const tpools = mixedRoute.pools.map((tpool) => {
           if (tpool.protocol === Protocol.V2) {
             return PairMarshaller.unmarshal(tpool as MarshalledPair)
+          } else if (tpool.protocol === Protocol.V3) {
+            return V3PoolMarshaller.unmarshal(tpool as V3MarshalledPool)
           } else {
```
Comment (xrsv) [inline lib/handlers/marshalling/route-marshaller.ts:122]: i would an explicit check for V4 here, and throw exception on else, just like you did in other places

**Review by xrsv — APPROVED** [review]
looks good - left a couple of comments

**Pull request merged**